src/main/java/com/uber/jenkins/phabricator/conduit/HarbormasterClient.java

```diff
@@ -14,24 +15,32 @@
     public HarbormasterClient(ConduitAPIClient conduit) {
         this.conduit = conduit;
     }
+
     /**
      * Sets a sendHarbormasterMessage build status
      * @param phid Phabricator object ID
      * @param pass whether or not the build passed
      * @param unitResults the results from the unit tests
      * @param coverage the results from the coverage provider
+     * @param lintResults
      * @return the Conduit API response
      * @throws IOException if there is a network error talking to Conduit
      * @throws ConduitAPIException if any error is experienced talking to Conduit
      */
-    public JSONObject sendHarbormasterMessage(String phid, boolean pass, UnitResults unitResults, Map<String, String> coverage) throws ConduitAPIException, IOException {
+    public JSONObject sendHarbormasterMessage(String phid, boolean pass, UnitResults unitResults, Map<String, String> coverage, LintResults lintResults) throws ConduitAPIException, IOException {
 
         List<JSONObject> unit = new ArrayList<JSONObject>();
 
         if (unitResults != null) {
             unit.addAll(unitResults.toHarbormaster());
         }
 
+        List<JSONObject> lint = new ArrayList<JSONObject>();
+
+        if (lintResults != null) {
+            lint.addAll(lintResults.toHarbormaster());
```

Comment:
Sorry I missed some context here. Why not send an entire valid JSON to harbormaster, instead of splitting all the parameters for this sendHarbormasterMessage call.  Sent a review just in case.